paasta_tools/cli/cmds/status.py

```diff
@@ -246,8 +246,10 @@ def paasta_status_on_api_endpoint(
             cluster, service, instance, output, status.adhoc, verbose
         )
     elif status.flink is not None:
+        flink_status = status.flink.get("status")
+        flink_metadata = status.flink.get("metadata")
         return print_flink_status(
-            cluster, service, instance, output, status.flink.get("status"), verbose
```

Comment:
mb just pass `status.flink` into `print_flink_status()`, since the status to print is now more than just `status.flink.status`?

Reply:
I like this suggestion, thank you